src/stringBuffer.js

```diff
@@ -22,9 +22,18 @@
  * buffer() === 'The breakfast at 10AM'
  *
  * @return {function}
- */
+*/
+
 function makeBuffer() {
-  // write code here
+  let prev = '';
```

Comment:
prev? cache + string
```suggestion
  let cachedString = '';
```